pages/genes/[id].js

```diff
@@ -78,18 +79,9 @@ const Gene = ({ gene }) => {
             )}
             {gene.locations?.length > 0 && (
               <>
-                <DataItemLabel>Gene Locations</DataItemLabel>
+                <DataItemLabel>Gene GeneLocations</DataItemLabel>
```

Comment:
Why change "Gene Locations" to "Gene GeneLocations"?

Reply:
That’s a good question. Must have been a stray paste error.